js/app.js

```diff
@@ -0,0 +1,30 @@
+var toDoApp = angular.module('toDoApp', []);
+function ToDoController($scope) {
```

Comment:
Be consistent with spacing - if you use a space between `function` and `()`, do it consistently